File: business/src/main/java/org/helmut/profile/model/CvTO.java
package org.helmut.profile.model;

import java.util.List;

public class CvTO {
    private String about;
    private UserTO userTO;
    private List<ExperienceTO> experiences;
    private List<LanguageTO> languages;
    private List<CertificationTO> certifications;

    public UserTO getUserTO() {
        return userTO;
    }

    public void setUserTO(UserTO userTO) {
        this.userTO = userTO;
    }

    public List<ExperienceTO> getExperiences() {
        return experiences;
    }

    public void setExperiences(List<ExperienceTO> experiences) {
        this.experiences = experiences;
    }

    public String getAbout() {
        return about;
    }

    public void setAbout(String about) {
        this.about = about;
    }

    public List<LanguageTO> getLanguages() {
        return languages;
    }

    public void setLanguages(List<LanguageTO> languages) {
        this.languages = languages;
    }

    public List<CertificationTO> getCertifications() {
        return certifications;
    }

    public void setCertifications(List<CertificationTO> certifications) {
        this.certifications = certifications;
    }
}

File: web-client/src/main/js/profile/src/app/commons/model/to/cv-t-o.ts
import {UserTO} from "./user-t-o";
import {LanguageTO} from "./language-t-o";
import {ExperienceTO} from "./experience-t-o";
import {CertificationTO} from "./certification-t-o";

export interface CvTO {
  about: string;
  userTO: UserTO;
  languages: LanguageTO[];
  experiences: ExperienceTO[];
  certifications: CertificationTO[]
}
